components/list/demo/list-item-sample.js

```diff
@@ -0,0 +1,47 @@
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { ListItemCheckboxMixin } from '../list-item-checkbox-mixin.js';
+//import { ListItemDragMixin } from '../list-item-drag-mixin.js';
+
+class ListItemSample extends ListItemCheckboxMixin(LitElement) {
+	// TODO: Role and breakpoints to live elsewhere
+	static get properties() {
+		return {
+			href: { type: String }
+		};
+	}
+
+	// TODO: Placeholder grid styles to be replaced with list-item-generic
+	static get styles() {
+		return [ (super.styles ? super.styles : css``), css`
+			.list-grid {
+				display: grid;
+				grid-template-columns:
+					[start outside-control-start] 40px
+					[control-start outside-control-end] 40px
+					[control-end content-start] auto
+					[content-end actions-start] auto
+					[end actions-end];
+			}
+			.d2l-list-item-content {
+				grid-column: content-start / content-end;
+				grid-row: 1 / 2;
+			}
+			.d2l-list-item-actions {
+				grid-column: actions-start / actions-end;
+				grid-row: 1 / 2;
+			}
+		`];
+	}
+	render() {
+		return html`
+			<div class="list-grid">
+				${ this._renderDragHandle ? this._renderDragHandle() : '' }
+				${ this._renderCheckbox() }
+				<div class="d2l-list-item-content"></div>
```

Comment:
Are these `div`s meant to contain elements - i.e. they would either define a slot, or other elements?

Reply:
These dives are meant to define slots, which currently don't exist.

Reply:
I see.  Currently the `div[slot="control-action"]` element from the checkbox mixin is rendered on top of these (because it has `z-index: 2;`) so if these contained something interactive, these divs/slots, or the elements within would need a `z-index > 2` to put them on top of the stack.

Reply:
This is expected behaviour according to the design document. Anything interactive should be inside of one of the `action` slots or `actions`.

Reply:
I am a little confused by this, perhaps due to the names of the classes.  I think it might be worthwhile to see the sample with the slots and an action that works (on top of the clickable area for toggling the checkbox).

Reply:
Fair enough - I've altered the sample component to make this clearer and added a button to the `actions` slot.
![Screen Shot 2020-04-30 at 4 42 09 PM](https://user-images.githubusercontent.com/2412740/80757254-d7cd3700-8b01-11ea-9cb7-500bf61f682a.png)